rudder-webapp/SOURCES/rudder-webapp-postinst

```diff
@@ -90,28 +90,28 @@ fi
 # Need to restart to take schema changes into account
 echo -n "INFO: Restarting rudder-slapd..."
 
-# during upgrade, service may not have been started with systemctl
-# detect first if rudder-slapd is seen running ?
-RUDDER_SLAPD_RUNNING_SERVICE=0
-RUDDER_SLAPD_RUNNING_SYSTEMCTL=0
-if service rudder-slapd status > /dev/null; then 
-  RUDDER_SLAPD_RUNNING_SERVICE=1
-fi
-if systemctl is-active rudder-slapd > /dev/null; then 
-  RUDDER_SLAPD_RUNNING_SYSTEMCTL=1
-fi
-
-if [ $RUDDER_SLAPD_RUNNING_SERVICE -eq 1 ] && [ $RUDDER_SLAPD_RUNNING_SYSTEMCTL -eq 0 ]
-then
-  # systemctl doesn't know the service is started, we stop it with service first
-  service rudder-slapd stop >> ${LOG_FILE}
+if type service >/dev/null 2>&1; then
```

Comment:
Shouldn't we do something if it doesn't exist ?

Reply:
I don't know, because it is very unlike the service was started without systemd if the system does not have sysvinit installed